index.html

```diff
@@ -140,8 +154,10 @@
             .attr('value', function(d) { return d; })
             .html(function(d) { return d; });
         options.exit().remove();
-
-        update(indicators, range.value * 60);
+      }).on('select', function(e) {
+          $('#info').html(infoTemplate({data: e.data.structured}));
```

Comment:
I'll see about modifying the `pointset` endpoint to keep the additional, non-structured properties (store name and address, for this dataset) that are in the loaded `pointset` GeoJSON, but not in what the API returns.